### ISSUE TITLE
Support for other currencies 

### ISSUE THREAD

**Issue by l00pz123** [bitkey/bitkey#38]
Would it be possible to implement support for other currencies like ethereum, litecoin, monero, dash, ripple ?

**Comment by lirazsiri** [comment]
Yeah of course it should be. We just need to scout out the best open source
software for that. Would be interested in getting feedback on that from the
community. If I'm not mistaken creating paper wallets for all these
currencies is already supported.

On 7 Jun 2017 3:28 pm, "l00pz123" <notifications@github.com> wrote:

Would it be possible to implement support for other currencies like
ethereum, litecoin, monero, dash, ripple ?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/bitkey/bitkey/issues/38>, or mute the thread

**Comment by ghost** [comment]
There's already an electrum litecoin client - https://electrum-ltc.org/
Would that be easy to implement, leveraging the wrapper we already have now?

**Comment by l00pz123** [comment]
What about cooperation with coinomi? They are about to roll out their multi-currency wallet for linux.
It is extremely solid product.

**Comment by lirazsiri** [comment]
An open source multi currency wallet could be the way to go. Certainly
better than adding of adding a ton of different apps for each currency

**Comment by l00pz123** [comment]
you could take a look at latest beta:
https://beta.coinomi.com/desktopinstallers/Coinomi.deb 

Beta testers and the team: https://coinomi.slack.com

**Comment by lirazsiri** [comment]
Going to need to build this myself .  But I can't find the source code for
the desktop version. Could you send me a link to that?

**Comment by jimbursch** [comment]
If I can get funding for you, can you build a version of Bitkey for Dash? Or, if you are building a multi-currency version, would it help if you got funding?

**Comment by lirazsiri** [comment]
Whether or not Dash is included in a multi currency version primarily
depends on the support in the underlying tools. We'd have to take a look at
what the state of that is to decide whether it makes sense to include Dash
in a multi currency version or do a Bitkey version focused on Dash and
related tools.

Anyhow, if you could can get funding, we would post bounties to develop a
Bitkey version for Dash. Or you could do that on the Dash side.

**Comment by DFF-fred** [comment]
interested in Bitkey for other things than Bitcoin I was looking at Coinomi and found this https://github.com/Coinomi/coinomi.github.io/issues/5 which indirectly says it is not open source.

**Comment by JedMeister** [comment]
Thanks @DFF-fred - I just dug a bit deeper and I think I've found their licence. It's called the "Coinomi Software License" and the news for BitKey isn't good:

> solely for personal, internal, non-commercial purposes and shall not distribute, sell or transfer the Software or Improvements to any person or third parties without prior written permission from the Licensor

source: https://github.com/Coinomi/coinomi-android/blob/master/COPYING

So it seems clear that the only way that we could legally include Coinomi is if end users build it themselves (with Coinimi downloaded and installed at build time). Although, that would be possible, it's hardly user friendly. 

So someone such as @l00pz123 could fork BitKey and include Coinomi, but the pre-built ISO could not be redistributed. 

I guess we could ask for a licence to redistribute, but personally I'm quite prejudice against non-free (as in freedom) software. This is actually a good example of the difference between "Open Source" and "Free" software. In this case it's not quite as bad as run-of-the-mill proprietary software as at least the source code is visible so it's essentially "open source". But as the end user's rights are limited, it's not truly "Free" software.

**Comment by jimbursch** [comment]
I've started a discussion at Dash Forum about funding a Dash version of BitKey:
https://www.dash.org/forum/threads/poor-mans-hardware-wallet-bitkey-port-to-dash.16150/

There seems to be support for this idea. It was suggested by @lirazsiri that a bounty be posted to develop a Dash version. What would be a good amount for a bounty? In addition to funding the dev, we would also fund ongoing support.

**Comment by daniel-lima** [comment]
It's far from being the full requested feature, but a few days ago I added some cold wallet/paper wallet generators: https://github.com/bitkey/bitkey/pull/42

Regards,

Daniel.

**Comment by itsjoelee** [comment]
Hi, could someone please instruct me on how to use BitKey app on my iPhone. I downloaded and installed the app, I sent my bitcoin to the address of the wallet, now I can see the amount of bitcoin I have in there but I don't know how to move it to another address. Could someone please give me instructions on how I would go about this? Thanks

**Comment by secme** [comment]
How about the Encompass electrum fork, seems like it'd fit right in; https://github.com/mazaclub/encompass

**Comment by DFF-fred** [comment]
Hi again! While encompass seemed like a nice option, this time it just seems it is no longer maintained and would require too much work to bring it into a usable state. See https://github.com/mazaclub/encompass/issues/152#issuecomment-304510839 for more details.

**Comment by isvekris** [comment]
Hey guys, 
Did you got around implementing etherum wallet? :)

**Comment by AndreasBBS** [comment]
Maybe this https://github.com/ExodusMovement is possible to integrate. Thoughts?

**Comment by secme** [comment]
Exodus is closed source. I've got the offline version of https://www.myetherwallet.com/ working from a usb key, and use that for multi-sig. Just need to integrate that and Job done, you support all ERC20 tokens.

**Comment by jologi** [comment]
Any news on that subject? Which currencies are currently supported?

**Comment by jluttine** [comment]
I created a very simple live USB OS with support for a few other cryptos. Should be relatively easy to add support for some other popular cryptos too. It's based on NixOS and very much work in progress, so I can't advice anyone to use it. But just thought I'd mention it here in case someone is interested: https://github.com/jluttine/CryptOS. Obviously, it's not as feature rich and polished as Bitkey.

**Comment by estevaocm** [comment]
Ethereum, litecoin, monero, dash and ripple as requested by the opening post are now supported in the community branch.
